### DISCUSSION TITLE
How can I apply the json file created from gradience on to debian12(bookworm) with gnome desktop?

### DISCUSSION THREAD

**Discussion by yichuang-zh** [GradienceTeam/Gradience#766] [Q&A] [answered]
I want to apply the json file made by gradience onto debian bookworm, but I don't know how to apply it.

ps:
There is no gradience installed on debian bookworm.

**Comment by 0xMRTT** [comment, marked as answer]
You can install Gradience from Flatpak and use it to apply

**Reply by yichuang-zh** [reply]
So the theme effects created with gradience can not be used on systems that don't have gradience installed?

This is weird.
I think it's because of gnome, not gradience. Maybe gnome is not as well as they said.

**Reply by daudix** [reply]
Well, this is file generated by gradience for gradience, if you want to save theme to be used without gradience you should copy "compiled" css from `~/.config/gtk-4.0/`. This is not monopoly

**Reply by tfuxu** [reply]
Yes, that JSON file is just a preset configuration, from which Gradience generates a `gtk.css` file that can be used by Libadwaita. Basically if you want to apply your preset, you need to install Gradience, import that preset and click Apply in upper right in order to generate and load a CSS stylesheet that can be understood by GTK/Libadwaita.

**Reply by yichuang-zh** [reply]
Thanks~

I had installed gradience flatpak on my debian and it works well.
